cos-event/run

```diff
@@ -0,0 +1,123 @@
+#!/bin/bash
+
+# Env Vars:
+# REPOSITORY: name of the image registry/namespace to get the images
+# COS_ID: If set, specifies the full CRN of a Cloud Object Storage instance to use
+
+# Use a dynamic bucket name because COS prevents bucket name reuse for
+# 15 minutes after bucket deletion
+function bucket_name() {
+  set +x
+  PROJECT=$1
+  BASE=$(ibmcloud ce project get --name ${PROJECT}| awk '/^ID/{ print $2 }')
+  EPOCH=$(date +%s)
+  echo "${BASE}-${EPOCH}"
+  set -x
+}
+
+# Clean up previous run
+function clean() {
+  set +x
+  DCOS=$1
```

Comment:
I think if we accept the fact that we're going to use a non-random bucket name, then a think a lot of the conditional logic in here can go away

Reply:
Agreed.